include/constants/event_objects.h

```diff
@@ -295,6 +295,47 @@
 
 #define SHADOW_SIZE_XL_BATTLE_ONLY  SHADOW_SIZE_NONE    // Battle-only definition for XL shadow size.
 
+// If true, adds a small amount of overhead
+// to OW code so that large (48x48, 64x64) OWs
+// will display correctly under bridges, etc.
+#define LARGE_OW_SUPPORT TRUE
```

Comment:
All new configs should be moved to `include/config/overworld.h`.
Also, we already have configs such as `OW_LARGE_OW_SUPPORT`, so incoming code should be modified to use them.